core/core-impl/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/move/generic/list/kopt/FlipSublistAction.java

```diff
@@ -31,7 +31,7 @@ public KOptAffectedElements getAffectedElements() {
     }
 
     void doMoveOnGenuineVariables() {
-        flipSublist(combinedList, fromIndexInclusive, toIndexExclusive);
+        flipSublist(combinedList, 0, fromIndexInclusive, toIndexExclusive);
```

Comment:
Dtto.
How is 0 safe here?

Reply:
Because it is an unused method (or should be unused), will remove code

Reply:
Note: where is this method, I could not find the overload (only `doMoveOnGenuineVariables(MultipleDelegateList<?> combinedList)` (which takes an arg), which I added a comment to)